tests/e2e/e2e_test.go

```diff
@@ -165,8 +161,7 @@ func (s *IntegrationTestSuite) TestIBCTokenTransferRateLimiting() {
 	fmt.Println(wd, projectDir)
 	err = copyFile(projectDir+"/x/ibc-rate-limit/bytecode/rate_limiter.wasm", wd+"/scripts/rate_limiter.wasm")
 	s.NoError(err)
-	// set LatestCodeId to 1 since we upload a contract in the upgrade handler for v13
-	chainA.LatestCodeId = 1
+
 	node.StoreWasmCode("rate_limiter.wasm", initialization.ValidatorWalletName)
 	chainA.LatestCodeId += 1
```

Comment:
Just to make sure it's clear that this is being set to 1. Otherwise it's not immediately clear if we are incrementing something like 2 to 3
```suggestion
	chainA.LatestCodeId = 1
```